.github/workflows/check-and-assign-pr-to-ui-team.yml

```diff
@@ -0,0 +1,16 @@
+name: Check and Assign PR to UI team
+on:
+  pull_request:
+    types: [opened, reopened, synchronize]
+    paths:
+      # assign to UI team only if there is a change in a json schema/catalog or in the manager API
+      - 'processor/configuration/src/main/resources/schemas/*'
+      - 'openapi/smartevents_mgmt.yaml'
```

Comment:
The UI repository already has this check (but in [reverse](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox-ui/blob/main/.github/workflows/openapi-sdk-compatibility-check.yml)).

Reply:
But this does no harm either :smile:

Reply:
IIUC with that check the UI team is "notified" about the changes only after we have merged something. Watching the openapi file here we ensure that they are always notified if any change is going to be applied in the API, which is good